geode-assembly/src/distributedTest/java/org/apache/geode/session/tests/GenericAppServerContainer.java

```diff
@@ -154,8 +157,15 @@ private void modifyWarFile() throws IOException, InterruptedException {
     int exitCode = process.waitFor();
     // Throw error if bad exit
     if (exitCode != 0) {
-      throw new IOException("Unable to run modify_war script, command: " + builder.command()
-          + "\ncheck log file: " + modifyWarScriptLog.getAbsolutePath());
+      StringBuilder sb = new StringBuilder();
+      sb.append("Unable to run modify_war script, command: ").append(builder.command());
+      sb.append(lineSeparator());
+      sb.append("log file: ");
```

Comment:
Not sure if this is intentional but you lost the word "check" in this refactor.

Reply:
Not intentional. Pushing a fix to restore the lost word. Those \n make it too easy to delete a word when you get rid of \n.